### PR TITLE
Fabfile fixes based on user feedback

### PR DESCRIPTION
**Pull request by btidor**

### CODE REVIEW

**Comment by benjaminjkraft** [comment]
Looks fine to me, but I haven't set up fabric yet, so probably someone who has should at least glance at this.

File: fabfile.py [hunk 1]
```diff
@@ -89,7 +92,7 @@ def setup():
     print "***** Please choose a passphrase and type it at the three prompts."
```
Comment (mgersh) [inline fabfile.py]: I haven't tried it myself, but based on your commit message this should say "two prompts" now?

Comment (btidor) [inline fabfile.py]: Right, yes.

**Comment by mgersh** [comment]
Cool, looks good. Since it's a security fix, want to email out about it?

**Pull request merged**